### PR TITLE
Issue 131

### LINKED ISSUE #131: Move UAST switcher in Code modal to the header
It might be tricky.
On the first glance without redux we would need HOC for the modal component to implement that.

### PR DESCRIPTION
**Pull request by carlosms**
Fixes #131.

It might be a good idea to do a similar refactor to `showUAST`, but it is unrelated and can be done when the code viewer is finished.

a

### CODE REVIEW

File: frontend/src/components/CodeViewer.js [hunk 1]
```diff
@@ -157,48 +137,73 @@ class CodeViewer extends Component {
...
+            CODE
+            <Switch
+              checked={showUast}
+              onChange={this.handleShowUastChange}
+              disabled={!language}
+              checkedIcon={<span className="switch-text checked">UAST</span>}
+              uncheckedIcon={
+                <span className="switch-text unchecked">UAST</span>
+              }
+              onColor="#ffba34"
```
Comment (smacker) [inline frontend/src/components/CodeViewer.js]: do you think we can move it to css?

some class with `!important`.

File: frontend/src/components/CodeViewer.less [hunk 2]
```diff
@@ -21,3 +23,22 @@
...
+.code-toggler {
+    margin-left: 40px;
+
+    .switch-text {
+        font-size: 18px;
+        font-weight: @btn-font-weight;
+    }
+
+    .switch-text.checked {
+        color:    @btn-gbpl-tertiary-color;
```
Comment (smacker) [inline frontend/src/components/CodeViewer.less]: extra spaces

**Comment by carlosms** [comment]
Rebased to fix conflicts.

I agree with the hardcoded colors in JS, but it's the only supported way by `react-switch`. I tried to move it to css in 42414e4, please take another look.

**Comment by smacker** [comment]
thanks! yes. I prefer this `!important` override.

**Pull request merged**